server.go

```diff
@@ -1944,6 +1961,40 @@ func SendHeader(ctx context.Context, md metadata.MD) error {
 	return nil
 }
 
+// SetSendCompressor sets a compressor for outbound messages.
+// It must not be called after any event that causes headers to be sent
+// (see SetHeader for a complete list). Provided compressor is used when below
+// conditions are met:
+//
+//   - compressor is registered via encoding.RegisterCompressor
+//   - compressor name exists in the client advertised compressor names sent in
+//     :grpc-accept-encoding header.
```

Comment:
Nit: remove the leading colon; this is not a pseudo-header and does not contain a leading colon.